pages/Threat_Modeling_Outputs.md

```diff
@@ -0,0 +1,61 @@
+---
+
+layout: col-sidebar
+title: Threat Modeling Outputs
+author: Adamshostack
+contributors: Avid, irene221b, lojikil, mplatt, Spartucvs, swierckx, tash,
+    tonyuv, zeroxten, ADubhlaoich
+permalink: /Threat_Modeling_Output
+tags: 'Threat Modeling'
+
+---
+
```

Comment:
If you wanted to have the author and contributors displayed on page you could do something like:

```html
Author: {{ page.author }} 
<br>
Contributors: {{ page.contributors }}
```

Below the front matter and before the first heading. In the other doc it renders like this:
![image](https://user-images.githubusercontent.com/7570458/77271136-45c14b80-6c84-11ea-8fcd-3fba683fc463.png)

Reply:
It doesn't actually matter that much to me personally - I added the contributors for this page and the previous page from the wiki page history; if the wiki is eventually taken offline there would be no record visible on the page or in the metadata of contributors otherwise.

I have no intention of adding the contributors directly onto the page, as that would need to be done to literally every other page to make sure it had a consistent layout - if whoever runs the website or the website the template is derived from (I'm not 100% sure what the politics there are) wants the people working on pages publicly visible they should add it to the actual page template, that way it can be automatically derived from the metadata.

Reply:
That's fair.